docs/source/custom_sockets.rst

```diff
@@ -0,0 +1,214 @@
+.. _custom-sockets:
+
+=========================
+Custom Sockets in Syncano
+=========================
+
+``Syncano`` provides possibility of creating the custom sockets. It means that there's a possibility
+to define a very specific endpoints in syncano application and use them as normal api calls.
```

Comment:
I think we should use API not api to be more correct here (with it being an acronym and all). Same for a few other uses below.